gen/gen.go

```diff
@@ -334,35 +294,14 @@ func (g *Generator) resolveLogicalSchema(logicalType avro.LogicalType) string {
 	return typ
 }
 
-func (g *Generator) newField(name, typ, tag string) field {
-	tagLine := fmt.Sprintf(`avro:"%s"`, tag)
-	for tagName, style := range g.tags {
-		if tagName == "avro" {
-			continue
-		}
-		tagLine += fmt.Sprintf(` %s:"%s"`, tagName, formatTag(tag, style))
-	}
+func (g *Generator) newField(name, typ, avroFieldDoc, avroFieldName string) field {
+	delete(g.tags, "avro")
```

Comment:
Perhaps this could be done earlier, like in the constructor?

Reply:
moved to constructor :) -> most likely this "avro" constant needs even more work